packages/file-settings/README.md

```diff
@@ -133,3 +163,39 @@ const settings: Settings = { ... };
 ### `Settings<T = Package>`
```

Comment:
We can remove this section as well. There's not much that users should look for here.